src/main/java/com/example/calculator/GlobalExceptionService.java

```diff
@@ -9,9 +9,8 @@
 @ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
т.к. контроллер с аннотацией `@RestController`